src/github/client/github-client.test.ts

```diff
@@ -50,13 +50,13 @@ describe("GitHub Client", () => {
 		expect(outboundProxyConfig.httpsAgent).toBeUndefined();
 	});
 
-	describe('config object', () => {
-		const TEST_API_URL = 'http://api.myBaseUrl.com';
-		const TEST_GRAPHQL_URL = 'http://graphql.myBaseUrl.com';
+	describe("config object", () => {
```

Comment:
I do this before commit `yarn lint:ts --fix`, not sure why there was `'`, but seems the linter always change it to `"`